src/SldStyleParser.ts

```diff
@@ -169,18 +193,17 @@ class SldStyleParser implements StyleParser {
    * @return {ScaleDenominator} The GeoStyler-Style ScaleDenominator
    */
   getScaleDenominatorFromRule(sldRule: any): ScaleDenominator | undefined {
-    let min: number | undefined;
-    let max: number | undefined;
+    let scaleDenominator: ScaleDenominator = <ScaleDenominator> {};
     if (sldRule.MinScaleDenominator) {
-      min = parseFloat(sldRule.MinScaleDenominator[0]);
```

Comment:
Is a ScaleDenominator a float value or more an integer? Not sure what the SLD spec says.

Reply:
SLD spec defines it as double so i'll stick with float.